src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs

```diff
@@ -70,4 +72,60 @@ public static IReviewConditionsBuilder<TReviewSettings> MinimumTimeElapsedSinceA
 		);
 		return builder;
 	}
+
+	/// <summary>
+	/// Add a custom synchronous condition.
+	/// </summary>
+	/// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+	/// <param name="builder">The builder.</param>
+	/// <param name="condition">The condition.</param>
+	/// <returns><see cref="IReviewConditionsBuilder{TReviewSettings}"/>.</returns>
+	public static IReviewConditionsBuilder<TReviewSettings> WithSynchronousCondition<TReviewSettings>(this IReviewConditionsBuilder<TReviewSettings> builder, SynchronousReviewCondition<TReviewSettings> condition)
```

Comment:
I don't think the overloads where you pass a condition object are going to be useful. People can already add any `IReviewCondition` so this seems like a less flexible option.

Reply:
That's right the container is public. My mistake.